test/components/Keystore.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import { ICGKeystore } from '../../src/renderer/services/interface';
+import { Eth1ICGKeystore } from '../../src/renderer/services/Eth1ICGKeystore';
+import { PrivateKey } from '@chainsafe/bls/lib/privateKey';
+import { PublicKey } from '@chainsafe/bls/lib/publicKey';
+import { Keypair } from '@chainsafe/bls/lib/keypair';
+const eth1WalletProvider = require('ethereumjs-wallet');
+import * as fs from 'fs';
+
+const privateKey = '0e43429c844ccedd4aff7aaa05fe996f41f9464b360ca03a4349387ba49b3e18';
+const privateKeyStr = `0x${privateKey}`;
+
+const keyStoreFilePath = `test/components/${getV3Filename()}.json`;
+const password = 'test';
+const newPassword = 'newTest';
+
+function getV3Filename(timestamp?: number) {
+    const ts = timestamp ? new Date(timestamp) : new Date();
+    return ['UTC--', ts.toJSON().replace(/:/g, '-'), '--', 'uuid'].join('');
```

Comment:
you should mock fs methods. Something like
const writeStub = sandbox.stub(fs, "writeFileSync")